src/components/search-form/search-form.js

```diff
@@ -229,6 +258,77 @@ class SearchForm extends Component {
       );
   }
 
+  renderAccessTypesFilter() {
+    const {
+      accessTypesStoreData,
+      searchByAccessTypesEnabled,
+      searchFilter = {},
+    } = this.props;
+
+    const {
+      'access-type': accessTypes = ''
+    } = searchFilter;
+
+    const {
+      sections,
+    } = this.state;
+
+    const accessTypesList = accessTypes ? accessTypes.split(',').map(accessType => {
+      return accessType.toLowerCase();
+    }) : [];
+
+    accessTypesList.sort();
+
+    const accessStatusTypesExist = !!accessTypesStoreData?.items?.data?.length;
+
+    return accessTypesStoreData?.isLoading
+      ? <Icon icon="spinner-ellipsis" />
+      : accessStatusTypesExist && (
```

Comment:
Please change `&&` to ternary operator with null. If `accessStatusTypesExist` is `false` we will render `0`

Reply:
`false` won't be rendered as `0`

Reply:
By same implementation we have this [bug](https://issues.folio.org/browse/UIEH-859)

Reply:
@Vladyslav-Velytskyi, we probably have it because we forgot to transform a number value into a boolean value (`{something.length && ...}` should be `{!!something.length && ...}`). `false` is not rendered as `0`. Also, I'm not able to reproduce this bug.